app/models/pglogical_subscription.rb

```diff
@@ -143,17 +136,17 @@ def self.dsn_attributes(dsn)
   end
   private_class_method :dsn_attributes
 
-  def self.provider_node_attributes(node_name)
+  def self.remote_region_attributes(subscription_name)
     attrs = {}
-    attrs[:provider_region] = MiqPglogical.node_name_to_region(node_name)
+    attrs[:provider_region] = subscription_name.split("_")[1].to_i
     region = MiqRegion.find_by_region(attrs[:provider_region])
     attrs[:provider_region_name] = region.description if region
     attrs
   end
-  private_class_method :provider_node_attributes
+  private_class_method :remote_region_attributes
 
   def self.subscriptions
-    pglogical.enabled? ? pglogical.subscriptions : []
+    pglogical.subscriptions
   end
   private_class_method :subscriptions
```

Comment:
At this point we may just want to do

```ruby
class << self
  delegate :subscriptions, :to => :pglogical
end
```

Reply:
Right now it looks like it's the only one with that direct of a correlation.

Reply:
Also I can't do that now that I needed to pass the database name, so I'm gonna resolve this one.